en/02_Developer_Guides/02_Controllers/02_Routing.md

```diff
@@ -173,99 +257,135 @@ Alternatively, if access to the parameters is not required in this way then it i
 URL parameters but not collect them in the same way:
 
 ```php
-<?php
-class StaffController extends \SilverStripe\Control\Controller
+namespace App\Control;
+
+use SilverStripe\Control\Controller;
+use SilverStripe\Control\HTTPRequest;
+
+class StaffController extends Controller
 {
     private static $url_handlers = [
-        'staff/$*' => 'index',
+        '$*' => 'index',
```

Comment:
If we're removing the leading `staff/` then we'd also need

```
private static $url_segment = 'staff';
```
?

Reply:
`$url_segment` is not used to determine routing rules at this level - it's only used for the following scenarios:
- Routing of `LeftAndMain` subclasses, for CMS director-level routing rules specifically (which is probably beyond the scope of this doc section)
- For generating a `Link()` method result in any `RequestHandler`

Controllers don't use it to determine which action should be used. For anything that isn't a `LeftAndMain` it's entirely optional - it's best practice but not necessary to declare it.

Reply:
Oh right sorry, I read this out of context, the yml above routes staff to the staff controller.